content/en/blog/2023/traffic-for-ambient-and-sidecar/index.md

```diff
@@ -0,0 +1,105 @@
+
+There are many blogs to introduce and analyze ambient in community and technology forums, and this blog will analyze the network traffic path in Istio ambient and sidecar modes. We will analyze the network traffic path between services in these two modes.
+
+In order to clarify the network traffic paths, this blog will explore two concrete scenarios with corresponding diagram to make it easier to understand. These two scenarios include:
+
+- **The network path of services in ambient mode to services in sidecar mode**
+- **The network path of services in sidecar mode to services in ambient mode**
+
+_Note 1: The following analysis is based on Istio 1.18.2, where ambient mode uses iptables for redirection._
+
+_Note 2: The communications between sidecar and ztunnel/waypoint proxy uses `[HTTP Based Overlay Network (HBONE)](https://docs.google.com/document/d/1Ofqtxqzk-c_wn0EgAXjaJXDHB9KhDuLe-W3YGG67Y8g/edit)`._
+
+## Ambient sleep to sidecar httpbin
+
+### Deployment and configuration for the first scenario
+
+- sleep is deployed in namespace foo
+    - sleep pod is scheduled to Node A
+- httpbin is deployed in namespace bar
+    - httpbin is scheduled to Node B
+- foo namespace enables ambient mode (foo namespace contains label: `istio.io/dataplane-mode=ambient`)
+- bar namespace enables sidecar injection (bar namespace contains label: `istio-injection: enabled`)
+
+With the above description, the deployment and network traffic paths are:
+
+{{< image width="100%"
+    link="ambient-to-sidecar.png"
+    caption="Ambient sleep to Sidecar httpbin"
+    >}}
+
+`ztunnel` will be deployed as a DaemonSet in istio-system namespace if ambient is enabled, while istio-cni and ztunnel would generate iptables rules and routes for both the ztunnel pod and pods on each node.
+
+All network traffic coming in/out of the pod with ambient enabled will go through ztunnel based on the network the redirection logic. The ztunnel will then forward the traffic to the correct endpoints.
+
+### sleep -> ztunnel -> [ sidecar -> httpbin ] network traffic path
+
+According to above diagram, the details of network traffic path is demonstrated as below:
+
+**(1) (2) (3)**  Request traffic of service sleep is sent out from the `veth` of the sleep pod, then it will be marked and forwarded to the `istioout` device in the node by following the iptables rules and route rules. The `istioout` device in the node A is a `[geneve](https://www.rfc-editor.org/rfc/rfc8926.html)` tunnel, and the other end of the tunnel is `pistioout`, which is inside the ztunnel pod on the same node.
+
+**(4) (5)**  When the traffic arrives `pistioout` device, it will be intercepted through the interface `eth0` of the ztunnel pod on port 15001 by iptables rules inside the pod.
```

Comment:
```suggestion
**(4) (5)**  When the traffic arrives through the `pistioout` device, it will be intercepted and redirected through the interface `eth0` of the ztunnel pod on port 15001 by iptables rules inside the pod.
```

Reply:
okay, already done!